internal/common/db.go

```diff
@@ -194,6 +194,16 @@ func GetHostFromDB(db *gorm.DB, clusterId, hostId string) (*Host, error) {
 	return &host, nil
 }
 
+func GetV2HostFromDB(db *gorm.DB, infraEnvId, hostId string) (*Host, error) {
```

Comment:
no need to rename this function to GetV2HostFromDB, it is a core function, starting from this point we always get host using InfraEnvID

Reply:
We still have the old one

Reply:
And it is using cluster id